libraries/plugins/peerplays_sidechain/addresses.txt

```diff
@@ -0,0 +1,391 @@
+2N5aFW5WFaYZLuJWx9RGziHBdEMj9Zf8s3J
```

Comment:
What is the purpose of this file ?

Reply:
Just the record of addresses I am using, its accidentaly added. I will remove it.